.github/workflows/bin-solr-test.yml

```diff
@@ -18,6 +18,10 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 40
 
+    strategy:
```

Comment:
IMO we should leave multi-JDK testing to Jenkins, and keep these GHA CI for PRs where we don't need to test multi-JDK.  The likelihood of breaking another JDK is small, and wouldn't be specific to the bats tests.